code/modules/modular_computers/file_system/programs/nttransfer.dm

```diff
@@ -170,7 +170,7 @@ var/global/nttransfer_uid = 0
 		data["uploading"] = 1
 		data["upload_uid"] = unique_token
 		data["upload_clients"] = connected_clients.len
-		data["upload_haspassword"] = server_password ? 1 : 0
+		data["haspassword"] = server_password ? 1 : 0
```

Comment:
Grumble. Grumble. This really should be a change to the template instead of the ui_data but I'll do it post freeze.